packages/uswds-core/src/styles/variables/project-colors.scss

```diff
@@ -94,6 +94,9 @@ $project-state-colors: (
     "light": set-theme-color(settings.$theme-color-disabled-light, no-warn),
     "default": set-theme-color(settings.$theme-color-disabled, no-warn),
     "dark": set-theme-color(settings.$theme-color-disabled-dark, no-warn),
+    "text": set-theme-color(settings.$theme-color-disabled-text, no-warn),
+    "text-reverse":
+      set-theme-color(settings.$theme-color-disabled-text-reverse, no-warn),
```

Comment:
> **Warning**
> I could not add "text-on-disabled" here because it caused an error for not following the standard `color`-`grade` naming format. There are a couple of paths forward here if we want to make this consistent. 
> 
> One option would be to give `$theme-color-text-on-disabled` (and maybe the other new settings) a name that fits better with the current state token format. (For this example, we could potentially appropriate `disabled-dark` because it is no longer used.) 
>
> Alternatively, I wonder if these new disabled settings might be better described as component tokens since they deviate in both purpose and format from other state colors. 
>
> Curious to hear what you think.